tools/topology/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ set(TPLGS
 	"sof-imx8qxp-wm8960\;sof-imx8qxp-wm8960"
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682"
 	"sof-tgl-max98373-rt5682\;sof-tgl-max98373-rt5682"
+	"sof-tgl-sdw-max98373-rt5682\;sof-tgl-sdw-max98373-rt5682\;-DSDW=1"
```

Comment:
can we define SDW in the .m4 file as we don't need this to be configurable at compile time?